tests/components/myuplink/test_init_2.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for init module - alternate fixtures."""
+from unittest.mock import MagicMock
+
+import pytest
+
+from homeassistant.components.myuplink.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+@pytest.fixture
```

Comment:
@MartinHjelmare helped out to setup the fixture structure in conftest.py for some time ago. Now it is time to introduce tests that need to override the default fixtures for API responses. I cannot make it work unless the alternate tests are moved to a separate module. All help to put them back in test_init.py is appreciated. Or perhaps it is nice and tidy too keep them in a separate module?

Reply:
Use the `pytest.mark.parametrize` decorator to override this fixture return value.